Tournament Content Usage Update (#11271)
ppy/osu-wiki: wiki/Tournaments/Official_support/en.md

- The forum threads associated with the tournament — this includes forum threads for any preliminary events — **MUST** contain a clearly visible link in a normal-sized font to the [tournament reports form](https://pif.ephemeral.ink/tournament-reports) as the very last content of the original post.
  - `https://pif.ephemeral.ink/tournament-reports`
  - This report form is overseen by the [Tournament Committee](/wiki/People/Tournament_Committee). We encourage all users — players and staff alike — to make use of this form where necessary. Any breaches of [expectations](#expectations), [other procedures](#other-procedures), or [eligibility](#eligibility) requirements should prompt a report.

> - The tournament must only use beatmaps of songs which abide by the [Content Usage Permissions](/wiki/Rules/Content_usage_permissions#artist-permissions). Any usage of tracks which are explicitly disallowed will revoke a tournament's eligibility for support. See the [Ranking Criteria](/wiki/Ranking_criteria#rules.4) for details regarding exceptions for artists or tracks which are listed as disallowed on the Content Usage Permissions.

Reviewer: same thing as https://github.com/ppy/osu-wiki/pull/11262#discussion_r1569114353

```suggestion
- The tournament must only use beatmaps of songs which are cleared in terms of [content usage permissions](/wiki/Rules/Content_usage_permissions#artist-permissions). Any usage of tracks which are explicitly disallowed will revoke a tournament's eligibility for support. See the [Ranking Criteria](/wiki/Ranking_criteria#rules.4) for details regarding exceptions for artists or tracks which are listed as disallowed on the Content Usage Permissions.
```

Author: Applied
